tests/UI/campaigns/functional/BO/14_advancedParameters/08_logs/01_filterSortAndPagination.ts

```diff
@@ -39,6 +29,11 @@ Sort logs table by : Id, Employee, Severity, Message, Object type, Object ID, Er
  */
 
 describe('BO - Advanced Parameters - Logs : Filter, sort and pagination logs table', async () => {
+  let browserContext: BrowserContext;
+  let page: Page;
+  let numberOfLogs: number = 0;
+  const today: string = getDateFormat('mm/dd/yyyy');
```

Comment:
This type can be inferred by `getDateFormat` directly, which should be enough

Reply:
Fixed